docs: adjust StandByCommittee node config documentation (#3375)
nspcc-dev/neo-go: docs/node-configuration.md

| P2PStateExchangeExtensions | `bool` | `false` | Enables the following P2P MPT state data exchange logic: <br>• `StateSyncInterval` protocol setting <br>• P2P commands `GetMPTDataCMD` and `MPTDataCMD` | Not supported by the C# node, thus may affect heterogeneous networks functionality. Can be supported either on MPT-complete node (`KeepOnlyLatestState`=`false`) or on light GC-enabled node (`RemoveUntraceableBlocks=true`) in which case `KeepOnlyLatestState` setting doesn't change the behavior, an appropriate set of MPTs is always stored (see `RemoveUntraceableBlocks`). |
| ReservedAttributes | `bool` | `false` | Allows to have reserved attributes range for experimental or private purposes. |
| SeedList | `[]string` | [] | List of initial nodes addresses used to establish connectivity. |

> | StandbyCommittee | `[]string` | [] | List of public keys of standby committee validators are chosen from. Note that it's not required to be sorted, but this list must have the same order of elements within configuration files of all the nodes in the network. |

Reviewer: Add it to the `Notes` section?

> | StandbyCommittee | `[]string` | [] | List of public keys of standby committee validators are chosen from. Note that it's not required to be sorted, but this list must have the same order of elements within configuration files of all the nodes in the network. |

Reviewer 2: ```suggestion
| StandbyCommittee | `[]string` | [] | List of public keys of standby committee validators are chosen from. Note that it's not required to be sorted, but the list must be exactly the same in configuration files of all the nodes in the network. |
```
is more straighforward imo
